Prox/Prox/Utilities/OpenInHelper.swift

```diff
@@ -33,6 +34,15 @@ struct OpenInHelper {
         return true
     }
 
+    private static func extractHostname(fromHost host: String) -> String? {
+        let components = host.components(separatedBy: ".")
+        if components.count > 0 {
+            return components[1]
```

Comment:
So these are always guaranteed to be `www` URLs? Could there be multiple sub-domains?